pennylane_lightning_gpu/src/util/CMakeLists.txt

```diff
@@ -3,4 +3,4 @@ set(CMAKE_CXX_STANDARD 20)
 
 add_library(lightning_gpu_utils INTERFACE)
 target_include_directories(lightning_gpu_utils INTERFACE $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
-target_link_libraries(lightning_gpu_utils INTERFACE CUDA::cublas)
+target_link_libraries(lightning_gpu_utils INTERFACE CUDA::cublas CUDA::cusparse)
```

Comment:
Since we are now linking against cusparse, we should also update https://github.com/PennyLaneAI/pennylane-lightning-gpu/blob/57a557e49639e78fea927eae714e28454cfea61b/docker/auditwheel#L11 to ensure we do not package it as part of the wheel build.